src/elements/common/annotator-context/withAnnotations.tsx

```diff
@@ -20,9 +21,7 @@ export default function withAnnotations<P extends object>(
         static displayName: string;
 
         state: AnnotatorState = {
-            annotation: undefined,
-            action: undefined,
-            error: undefined,
```

Comment:
Should these be removed? We are still saving the annotation payload here so the properties should still exist in the default state?

Reply:
Yeah, but they're all optional as @jstoffan pointed out, so removing them should be ok